indico/modules/events/registration/models/registrations.py

```diff
@@ -186,7 +186,11 @@ class Registration(db.Model):
         UTCDateTime,
         nullable=True
     )
-
+    #: If given a reason for rejection
+    rejection_reason = db.Column(
+        db.String,
+        nullable=True
```

Comment:
We usually use `nullable=False, default=''` to avoid a mix of `None` and `''` in the database.